test/uniqBy.js

```diff
@@ -0,0 +1,39 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('uniqBy', function() {
+
+  var objs = [
+    {i: 1, v: 1}, {i: 2, v: 2}, {i: 3, v: 3}, {i: 4, v: 4}, {i: 5, v: 5},
+    {i: 6, v: 6}, {i: 7, v: 7}, {i: 8, v: 8}, {i: 9, v: 9}, {i: 10, v: 10}
+  ];
+
+  var objs2 = [
+    {i: 1, v: 1}, {i: 2, v: 2}, {i: 2, v: 3}, {i: 4, v: 4}, {i: 5, v: 5},
+    {i: 6, v: 6}, {i: 6, v: 7}, {i: 6, v: 8}, {i: 9, v: 9}, {i: 5, v: 10}
+  ];
+
+  var getI = R.prop('i');
```

Comment:
I'd prefer to see `R.prop('i')` in place of `getI` wherever the latter is referenced. I believe tests should be as self-contained as possible, so a test can be read without one having to look for variable bindings in outer scopes.